pkg/util/constants.go

```diff
@@ -31,9 +31,19 @@ const (
 
 	AnnotationSkipRancherLoggingAddonWebhookCheck = prefix + "/skipRancherLoggingAddonWebhookCheck"
 
-	// AnnotationMigratingPrefix is used to store the migrating vm in the annotation of ResourceQuota
+	// AnnotationSkipResourceQuotaAutoScaling is used to disable to resourcequota auto scaling
+	AnnotationSkipResourceQuotaAutoScaling = prefix + "/skipResourceQuotaAutoScaling"
+
+	// AnnotationMigratingNamePrefix is used to store the migrating vm in the annotation of ResourceQuota
 	// eg: harvesterhci.io/migrating-vm1: jsonOfResourceList, harvesterhci.io/migrating-vm2: jsonOfResourceList
-	AnnotationMigratingPrefix = prefix + "/migrating-"
+	// will be removed after v1.5.0
+	AnnotationMigratingNamePrefix = prefix + "/migrating-"
+
+	// replace AnnotationMigratingNamePrefix from v1.5.0
+	AnnotationMigratingUIDPrefix = prefix + "/migratingUID-"
+
+	// AnnotationMigratingPrefix is replaced by AnnotationMigratingNamePrefix, and is kept for compatibility
+	AnnotationMigratingPrefix = AnnotationMigratingNamePrefix
```

Comment:
This variable is not referenced by any code. Do we need it?

Reply:
Harvester does not use it any more; but I did not check all others like lb, cloud-provider- ..., if any others refers it, directly remove const may break them.